spec/Section 6 -- Execution.md

```diff
@@ -534,9 +572,35 @@ DoesFragmentTypeApply(objectType, fragmentType):
     * if {objectType} is an implementation of {fragmentType}, return {true} otherwise return {false}.
   * If {fragmentType} is a Union:
     * if {objectType} is a possible type of {fragmentType}, return {true} otherwise return {false}.
+ 
```

Comment:
There is a space in this line that messes up building the spec.

When the spec is being build the space leads to the indentation of DeferFragment.

Reply:
<img width="185" alt="image" src="https://user-images.githubusercontent.com/9714350/93991304-69c52d00-fd8c-11ea-8537-a6cbbb5cba42.png">